jenkins/python-setup.sh

```diff
@@ -28,3 +28,9 @@ $(head -n 1 ${_prefix}/lib*/python3.*/site-packages/pbench.egg-link 2> /dev/null
 export PYTHONPATH
 
 python3 setup.py develop --prefix="${_prefix}"
+
+set -x
+ls -l $(realpath /dev/log)
```

Comment:
I might've added a `file $(realpath /dev/log)`

Reply:
Since this is a back-port, we might want to track such an addition on the `main` branch.

Reply:
This was debugging code which Dave commented on during the brief interval after it was added and before it was removed.  There is nothing more to see here, now.